eng/AfterSigning.targets

```diff
@@ -34,21 +34,21 @@
 
     <MakeDir Directories="$(ZipOutputDir)" />
     <Delete Files="%(LanguageServiceBinary.ZipFile)" />
-    
+
     <!-- Build a .zip file from each platform's directory and write it to 'artifacts' -->
-    
+
     <!-- Mac -->
     <Exec Command="pwsh -NonInteractive -command &quot;&amp;{ Write-Host &quot;Writing %(LanguageServiceBinary.ZipFile)...&quot; ; Add-Type -AssemblyName System.IO.Compression.FileSystem; [System.IO.Compression.ZipFile]::CreateFromDirectory('%(LanguageServiceBinary.SourceDir)', '%(LanguageServiceBinary.ZipFile)') }&quot;"
           Condition="$([MSBuild]::IsOSPlatform('OSX'))" />
-    
+
     <!-- Windows -->
     <Exec Command="powershell.exe -NonInteractive -command &quot;&amp;{ Write-Host &quot;Writing %(LanguageServiceBinary.ZipFile)...&quot; ; Add-Type -AssemblyName System.IO.Compression.FileSystem; [System.IO.Compression.ZipFile]::CreateFromDirectory('%(LanguageServiceBinary.SourceDir)', '%(LanguageServiceBinary.ZipFile)') }&quot;"
           Condition="$([MSBuild]::IsOSPlatform('Windows'))" />
-    
+
     <!-- Linux -->
-    <Exec Command="curl https://packages.microsoft.com/config/rhel/8/prod.repo | sudo tee /etc/yum.repos.d/microsoft.repo" Condition="$([MSBuild]::IsOSPlatform('Linux'))" />
-    <Exec Command="sudo yum install -y powershell 2>/dev/null" Condition="$([MSBuild]::IsOSPlatform('Linux'))" />
+    <Exec Command="curl https://packages.microsoft.com/config/rhel/8/prod.repo | sudo tee /etc/yum.repos.d/microsoft.repo" Condition="$([MSBuild]::IsOSPlatform('Linux')) AND '$(DotNetBuildFromSource)' != 'true'" />
```

Comment:
SourceBuild was publishing the Linux Razor language server, which we don't want due to duplicated assets.